.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1743,37 +1743,291 @@ public static class OpenAIFilesModelFactory {
     }
 }
 namespace OpenAI.FineTuning {
+    public readonly partial struct CreateFineTuningJobRequestModel : IEquatable<CreateFineTuningJobRequestModel> {
+        public CreateFineTuningJobRequestModel(string value);
+        public static CreateFineTuningJobRequestModel Babbage002 { get; }
+        public static CreateFineTuningJobRequestModel Davinci002 { get; }
+        public static CreateFineTuningJobRequestModel Gpt35Turbo { get; }
+        public static CreateFineTuningJobRequestModel Gpt4oMini { get; }
+        public readonly bool Equals(CreateFineTuningJobRequestModel other);
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override readonly bool Equals(object obj);
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override readonly int GetHashCode();
+        public static bool operator ==(CreateFineTuningJobRequestModel left, CreateFineTuningJobRequestModel right);
+        public static implicit operator CreateFineTuningJobRequestModel(string value);
+        public static bool operator !=(CreateFineTuningJobRequestModel left, CreateFineTuningJobRequestModel right);
+        public override readonly string ToString();
+    }
+    public class FineTuningCheckpoint : IJsonModel<FineTuningCheckpoint>, IPersistableModel<FineTuningCheckpoint> {
+        public string CheckpointId { get; }
```

Comment:
We can call this simply `Id`. Since the class is called `FineTuningCheckpoint`, it is clear that it is the checkpoint ID.

Reply:
Agreed. I think I misunderstood a conversation in a meeting instructing me to do the opposite. Will rename.